curriculum/challenges/english/16-the-odin-project/top-learn-variables-and-operators/learn-variables-and-operators-lesson-i.md

```diff
@@ -83,19 +83,20 @@ console.log( counter++ ); // 0
 
 ## --text--
 
-What is the difference in the result of the two console log statements in the JavaScript code below?
+What are the outputs of the two `console.log` statements in the JavaScript code below?
 
 ```js
 let counter = 1;
 console.log(2 * ++counter); // Statement A
-
+```
+```js
```

Comment:
````suggestion
```

```js
````

fenced code blocks always need empty lines around. Are you coding locally? this should be caught by the pre-commit validation